docusaurus.config.js

```diff
@@ -71,41 +71,37 @@ const config = {
       },
       navbar: {
         style: 'dark',
-        title: 'Nevermined Docs',
         logo: {
           alt: 'Nevermined Logo',
-          src: 'img/nvm-logo.png',
+          src: 'img/nevermined/nevermined_logo.svg',
         },
-        items: [        
+        items: [
           {
-            type: 'doc',
-            docId: 'getting-started/index',
```

Comment:
Links defined as `docId` are always active for some reason hence we cannot style the current active one.